components/recipes/modules/ROOT/pages/onboarding/sfuel/pow-distribution.adoc

```diff
@@ -0,0 +1,134 @@
+
+Distributing sFUEL through Proof-of-Work is a two-step process where the first step is optional depending on your SKALE Chain.
+
+== Step I | Deploy sFUEL Distribution Contract
+
+[NOTE]
+If you are building on a SKALE Hub or Chaos Testnet, checkout the xref:develop::sfuel/distribution/proof-of-work-distribution.adoc#_available_on_chain_faucets[deployed contracts] and skip to xref:_step_ii_setup_dapp[Step II].
+
+If you are on a dedicated SKALE Chain and have not deployed a sFUEL Distribution smart contract, yet, proceed with Step I.
+
+The following will use the link:https://github.com/Dirt-Road-Development/sfuel-distribution-contract[sFUEL Distribution Contract] by link:https://dirtorad.dev[Dirt Road Dev]. This contract is based on the example found xref:develop::proof-of-work.adoc[here] in the docs.
+
+=== Setup the Project
+
+Start by cloning (and optionally forking) the GitHub repository:
+
+```bash
+git clone https://github.com/Dirt-Road-Develompent/sfuel-distribution-contract && cd sfuel-distribution-contract
+```
+
+After entering into the root of the directory run the setup script:
+
+```bash
+chmod +x ./setup.sh && ./setup.sh
+```
+
+This will install npm packages, add the link:https://openzeppelin.com[Open Zeppelin] smart contracts package, and create a .env file in the root of project.
+
+=== Setup Environment
+
+The next step is to setup the environment. Open the .env file which looks like this:
+
+```.env
+# Private Key - DO NOT SHARE - Should NOT start with 0x
+DEPLOYER_PRIVATE_KEY=
+
+# SKALE Chain RPC
+RPC_URL=
+
+# Decimal Form
+CHAIN_ID= 
+
+# Blockscout -> Browser Url + /api
+API_URL=
+
+# Blockscout
+BROWSER_URL=
+```
+
+and fill in the all the values. The final result should look like this:
+
+```.env
+# Private Key - DO NOT SHARE - Should NOT start with 0x
+DEPLOYER_PRIVATE_KEY=0000000000000000000000000000000000000000000000000000000000000000
+
+# SKALE Chain RPC
+RPC_URL=https://mainnet.skalenodes.com/v1/some-amazing-schain-name
+
+# Decimal Form
+CHAIN_ID=11111111111
+
+# Blockscout -> Browser Url + /api
+API_URL=https://some-amazing-schain-name.explorer.mainnet.skalenodes.com/api
+
+# Blockscout
+BROWSER_URL=https://some-amazing-schain-name.explorer.mainnet.skalenodes.com
+```
+
+[WARNING]
+The private key added is linked to an Ethereum Address. That Ethereum Address must have Deployer Role -- see xref:develop::skale-chain-access-control.adoc#__whitelist[SKALE Chain Access Control] -- and at least as much sFUEL as defined in the link:https://github.com/Dirt-Road-Development/sfuel-distribution-contract/blob/25bd919f8c5edb15dccb5cb3b80d92f3e247101e/config.json#L5[deployment config here]. Deployment will fail without confirming these.
+
+[NOTE]
+If the deployer address does not have DEPLOYER_ROLE, the transaction will fail with an error on the explorer that says: +++<p style="color: red;">Error: Out of Gas</p>+++
+
+=== Deploy the Contract
+
+With everything setup, you can deploy the contract by running:
+
+[tabs]
```

Comment:
unfinished setup for tabs

Reply:
Resolved by e220a9b22cfccb25ff473b760f9ac54005eda6ae

**Note** - Removed table for simplicity at the current stage